panther_on_sonoma/compose.yaml

```diff
@@ -0,0 +1,51 @@
+x-common-config:
+  &common-config
+  network_mode: host
+  ipc: host
+
+x-cpu-config: &cpu-config
+  env_file: .env.cpu
+
+x-gpu-config: &gpu-config
+  runtime: nvidia
+  env_file: .env.gpu
+
+services:
+  panther_gazebo:
+    image: husarion/panther-gazebo:humble-on-sonoma
+    <<:
+      - *common-config
+      - *gpu-config
+    environment:
+      - PANTHER_IMU_ORIENTATION_R=0.0 # Moving on Sonoma caused orientation issue
+      - PANTHER_IMU_ORIENTATION_P=0.0
+      - PANTHER_IMU_ORIENTATION_Y=0.0
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix:rw
+      - ./config/components.yaml:/config/components.yaml
+    command: >
+      ros2 launch panther_gazebo simulation.launch.py
+        components_config_path:=/config/components.yaml
+        ekf_configuration:=global
+        gz_world:=/ros2_ws/src/husarion_gz_worlds/worlds/sonoma_raceway.sdf
+
+  foxglove:
+    image: husarion/foxglove:1.84.0
+    ports:
+      - 8080:8080
+      - 8765:8765
+    volumes:
+      - ./config/layout.json:/foxglove/default-layout.json
+    environment:
+      - DS_TYPE=foxglove-websocket
+      - DS_PORT=8765
+      - DS_HOST=foxglove-ds
+      - UI_PORT=8080
+      - DISABLE_CACHE=true
+      - DISABLE_INTERACTION=false
+
+  foxglove-ds:
+    image: husarion/foxglove-bridge:humble-0.7.3-20240108
+    command: ros2 launch foxglove_bridge foxglove_bridge_launch.xml
+      port:=8765
+      capabilities:=[clientPublish,connectionGraph,assets]
```

Comment:
As I recently developed lsc16 demo, rviz2 was used intentionally because of the licensing. I believe we should switch to rviz in this case as well.

Reply:
This image use open sourcevesion befor foxglove changing liense. It's shoud be ok, especially our webUI wisaso asedon the last open source release.